jellyfin_kodi/helper/playutils.py

```diff
@@ -384,14 +383,12 @@ def get_directplay_video_codec(self):
         return ",".join(codecs)
 
     def get_transcoding_video_codec(self):
-        codecs = ["h264", "hevc", "h265", "mpeg4", "mpeg2video", "vc1"]
+        codecs = ["h264", "mpeg4", "mpeg2video", "vc1"]
 
-        if settings("transcode_h265.bool"):
-            codecs.remove("hevc")
-            codecs.remove("h265")
-        else:
-            if settings("videoPreferredCodec") == "H265/HEVC":
-                codecs.insert(2, codecs.pop(codecs.index("h264")))
+        if settings("videoPreferredCodec") == "H265/HEVC":
+            codecs.insert(0, "hevc")
+        elif not settings("transcode_h265.bool"):
+            codecs.insert(1, "hevc")
```

Comment:
Does it matter what position this one is getting inserted too?  Unless I'm misunderstanding the logic, it could just be a simple `codecs.append` instead of manually specifying a position.

Reply:
My understanding is that the first position is used as the transcode target codec (if needed), so I'm treating it as a prioritized list of sorts (I have not actually tested that, just how I've read the code).

The tests that where added here test against the full return string – we could change that to split up the string, check the first position, and test if sets equal – but I haven't bothered doing that, hence the insert on the second one too.